examples/middleware/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "middleware",
+  "private": true,
+  "scripts": {
+    "dev": "next dev",
+    "build": "next build",
+    "start": "next start",
+    "lint": "next lint",
+    "build:worker": "pnpm cloudflare",
+    "dev:worker": "wrangler dev --port 8770 --inspector-port 9330",
```

Comment:
Can you please create an issue for that?
There should be no dependency here.

Reply:
> The other examples incremented the number to prevent two using the same, so I assume we want to do the same with new ones as well?

Yes, that's because we run the e2e tests in parallel and thus we don't want dev server to clash 👍

Reply:
yeah I agree, unfortunately that would require us to do some orchestration when running the tests... maybe it's worth investigating into it 👍